tag-pose/aruco_pose.py

```
@@ -0,0 +1,182 @@
+            calibration.getCameraIntrinsics(camera_socket, in_frame.getWidth(), in_frame.getHeight()),
+            dtype=np.float32,
+        )
+        distortion_coeffs = np.array(calibration.getDistortionCoefficients(camera_socket), dtype=np.float32)
+
+        corners_list, ids, _ = aruco_detector.detectMarkers(frame)
+
+        if ids is not None:
+            object_points = create_object_points(args.marker_size)
+            ids = ids.flatten()
+
+            for marker_id, corners in zip(ids, corners_list):
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)